oauth2_provider/models.py

```diff
@@ -62,8 +62,8 @@ class AbstractApplication(models.Model):
     RS256_ALGORITHM = "RS256"
     HS256_ALGORITHM = "HS256"
     ALGORITHM_TYPES = (
-        ("RS256", _("RSA with SHA-2 256")),
-        ("HS256", _("HMAC with SHA-2 256")),
+        (RS256_ALGORITHM, _("RSA with SHA-2 256")),
```

Comment:
IMHO, the `_ALGORITHM` part is not necessary cause model field let it explicit already.